suite-native/passphrase/src/PassphraseFormModal.tsx

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useRef } from 'react';
+import { Modal, TextInput } from 'react-native';
+
+import { useSetAtom } from 'jotai';
+
+import { Form, TextInputField, useForm } from '@suite-native/forms';
+import { yup } from '@trezor/validation';
+import { Box, Button, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useTranslate } from '@suite-native/intl';
+
+import { isPassphraseModalVisibleAtom } from './passphraseModalVisibilityAtom';
+
+const PASSPHRASE_MIN_LENGTH = 1;
+const PASSPHRASE_MAX_LENGTH = 50;
```

Comment:
Not related to this PR, but it would be nice to unify this with desktop, so we don't have to redefine this const. Particulary this file: https://github.com/trezor/trezor-suite/blob/7e0584c5112f414afef8c7c7616dd95cf007ba0a/packages/suite/src/constants/suite/inputs.ts

Reply:
I have an issue in the epic for moving thunks and validation consts to common - this will be fixed there. Tried to keep the diff maintainable for this one.